PrimeNumbers.py

```diff
@@ -0,0 +1,22 @@
+def is_prime(num):
+    if num <= 1: # 1 is not a prime number
+        return False
+    for div in range(2, num // 2 + 1): # loop to check range
+        if num % div == 0: # check condition if divided then prime
+            return False
+    return True
+
+
+def print_primes_in_loop(num):
+    for n in range(2, num + 1): # outer loop 
+       if is_prime(n):  # check if n is prime
+            print(n, end=" ")
```

Comment:
It looks like the nested ladder within the code that makes the code ugly. Try to write a code like a paragraph. How about this
```
def primes_in_range(num):
    for iterator in range(2, num + 1):

        if not is_prime(iterator):
            continue
        
        print(n, end=" ")
```